src/js/xr/src/use-interactions-manager.js

```diff
@@ -259,17 +258,35 @@ const useInteractionsManager = ({
 }) => {
   const { gl, camera, scene } = useThree()
 
+  
   const selections = useSelector(getSelections)
-
   const gpuPicker = useRef(null)
-
   const getGpuPicker = () => {
     if (gpuPicker.current === null) {
       gpuPicker.current = new GPUPicker(gl)
     }
     return gpuPicker.current
   }
 
+  useEffect(() => {
```

Comment:
Can you explain this a bit, maybe comment it? We need to create a fake object in the scene to initialize the picker?

Reply:
Basically, we need something to pick to initialize shader_log and stuff like that. If there's nothing in pick area(pick pixel) it won't initialize it. So I'm adding an object in front of the camera and picking it, that way there are definitely will be a pick and GPU picker will initialize properly.

Reply:
Could we move that code into a method of GPUPicker?

Reply:
I don't think GPU Picker needs to know about that stuff, it's kind of hack. I can move it to GPU picker but I think the cleaner way to do that is to keep it outside of it and use it as util or something